tests/tests_bond_nm.yml

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# This file was generated by ensure_provider_tests.py
+---
+# set network provider and gather facts
+- hosts: all
+  name: Run playbook 'playbooks/tests_bond.yml' with nm as provider
+  tasks:
+    - name: Set network provider to 'nm'
+      set_fact:
+        network_provider: nm
+      tags:
+        - always
+
+
+# workaround for: https://github.com/ansible/ansible/issues/27973
+# There is no way in Ansible to abort a playbook hosts with specific OS
+# releases Therefore we include the playbook with the tests only if the hosts
+# would support it.
+# The test requires or should run with NetworkManager, therefore it cannot run
+# on RHEL/CentOS 6
+- import_playbook: playbooks/tests_bond.yml
+  when:
+    - ansible_distribution_major_version != '6'
```

Comment:
No, it also runs on Fedora and there is no version "6", therefore != `6` means Fedora or RHEL/CentOS != `6` in this context.

Reply:
Ok.  So it implicitly skips this test on Fedora, rather than explicitly specifying that the test should be skipped on Fedora.
Is it possible that this role will someday support other platforms than EL and Fedora?

Reply:
It only skips it on CentOS/RHEL 6. I hope that by the time we support other plattforms that we do not need to support CentOS 6 anymore, so then this condition can just be removed.